### PR TITLE
mysql: bugfix for max connections [DOC-84]

### PR DESCRIPTION
**Pull request by wojcik-dorota**
Updated article `MySQL max_connections` to reflect our new way of defining the max number of connections for a service depending on a service plan/RAM

https://aiven.atlassian.net/browse/DOC-84

### CODE REVIEW

**Review by packi — APPROVED** [review]
LGTM, I would squash the commits though.

**Comment by lornajane** [comment]
Let me know if I can help with the squashing/rebasing/merging on this one?

File: .github/vale/styles/Aiven/capitalization_headings.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ exceptions:
   - Elasticsearch
   - European Union
   - Flink
+  - GiB
```
Comment (lornajane) [inline .github/vale/styles/Aiven/capitalization_headings.yml:39]: Could we double check the units intended here? GB would be gigabytes (which is what I expect, this is a standard measure), but GiB is for gibibytes ... is that correct?

Comment (wojcik-dorota) [reply .github/vale/styles/Aiven/capitalization_headings.yml:39]: @packi , could you please confirm?

Comment (packi) [reply .github/vale/styles/Aiven/capitalization_headings.yml:39]: We actually check for 4 GiB in the code

**Pull request merged**